_data/contributors.yml

```
@@ -21,7 +21,7 @@
 - name: Abdelatif Guettouche
   apacheId: aguettouche
   githubId: Ouss4
-  role: PPMC, Committer
+  role: PMC, PPMC, Committer
```

```suggestion
  role: PMC, Committer
```
PPMC is the podling PMC.  We are no longer a incubating "podling", we are a TLP (top-level project).  So any occurrences of PPMC should be replacement with PMC.  Same thing, just promoted to TLP. 

Same for the following roles that include both PMC and PPMC.  Those should be only PMC.

IPMC is something different.  You need to retain all references to the IPMC.  That is the incubator PMC that is in charge of all of the PPMCs.  So some people who were our mentors are in the IPMC and were also the PPMC (which is now the PMC).